Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -197,7 +198,8 @@ namespace O3DE::ProjectManager
 
             QMenu* menu = new QMenu(this);
             menu->addAction(tr("Edit Project Settings..."), this, [this]() { emit EditProject(m_projectInfo.m_path); });
-            menu->addAction(tr("Build"), this, [this]() { emit BuildProject(m_projectInfo); });
+            menu->addAction(tr("Auto-build"), this, [this]() { emit BuildProject(m_projectInfo); });
+            menu->addAction(tr("Open CMake..."), this, [this]() { emit OpenCMake(m_projectInfo); });
```

Comment:
Can this be "Open CMake-gui..."?

Reply:
I originally had "Open CMake GUI" but then realized it's just called CMake on Mac (cmake-gui on win/linux) and thought this might better indicated the intent of the engineer - to proceed with CMake manually instead of having the project manager automatically build everything for you.  This was also the recommendation from UX. Thoughts?

Reply:
its also called cmake-gui in `CMake.app/Contents/bin/cmake-gui` 
the "CMake" name in mac is just because that is the only application (the other ones are executables) and is in a different path. So, for bundles is a bit more convenient. However, in Windows/Linux that is not the case and its called cmake-gui.
Since "cmake" is the command-line tool, it may sound strange for those familiar with cmake
"Open CMake GUI" looks better to me.